network/src/commonMain/kotlin/com/wire/kalium/network/api/user/connection/ConnectionApi.kt

```diff
@@ -0,0 +1,27 @@
+package com.wire.kalium.network.api.user.connection
+
+import com.wire.kalium.network.utils.NetworkResponse
+import com.wire.kalium.network.utils.wrapKaliumResponse
+import io.ktor.client.HttpClient
+import io.ktor.client.request.post
+import io.ktor.client.request.setBody
+import kotlinx.serialization.json.buildJsonObject
+
+interface ConnectionApi {
+
+    suspend fun fetchSelfUserConnections(): NetworkResponse<ConnectionResponse>
+}
+
+class ConnectionApiImpl(private val httpClient: HttpClient) : ConnectionApi {
+
+    override suspend fun fetchSelfUserConnections(): NetworkResponse<ConnectionResponse> =
+        wrapKaliumResponse {
+            httpClient.post(PATH_CONNECTIONS) {
+                setBody(buildJsonObject {  })
```

Comment:
A quick question, looking at the swagger, this could be a request to get results paginated. Are we going to implement that later on? Or by design this is going to be ignored for the MVP ? So we can add just in case a `TODO` comment :D

Reply:
The paging will be implemented in the next PR